staff/daniel-ferrandez-latre/cv/index.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="es">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV Daniel FL</title>
+</head>
+<body>
+    <main>
+        <header>
+            <h1>Daniel Ferrández Latre</h1>
+        </header>
+        <section>
+            <article>
+                <section>
+                    <ul>
+                        <li>31 años</li>
+                        <li>
+                            <address>
+                                Mare de Deu de la Llum L'Hospitalet
+                            </address> 
+                        </li>
+                        <li>638078908</li>
+                        <li>daniel.sflatre@gmail.com</li>
+                    </ul>
+                </section>
+                <section>
+                    <picture>
+                        <img src="https://avatars2.githubusercontent.com/u/47394894?s=460&v=4" type="">
+                        </picture>
```

Comment:
@DaniLatreCodes  `</picture>`closing tag is not well indented. Please fix. Thanks.

Reply:
Done!